app/invocation/child_invocations.tsx

```diff
@@ -10,94 +9,51 @@ export type ChildInvocationProps = {
 };
 
 export default class ChildInvocations extends React.Component<ChildInvocationProps> {
+  private getDurationMillis(invocation: InvocationMetadata): number | undefined {
```

Comment:
Switch to just return 0 if not found and maybe rename to `getDurationMillisWithFallback` or something

Reply:
Thinking about it a bit more, I'd like to keep it as `number | undefined`  so that we can distinguish between a real duration == 0 (e.g. it failed after a few microseconds or something and got rounded down to 0) and something which doesn't have a duration (e.g. `queued` / `not-run` statuses).